README.markdown

```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
 
 ### close
 -----
-_**Description**_: Disconnects from the Redis instance, except when `pconnect` is used.
+_**Description**_: Disconnects from the Redis instance, include when `pconnect` is used.
```

Comment:
> include when

including when

Reply:
Alternatively, now that pconnect works also, you can remove the clause entirely. End result:

> Disconnects from the Redis instance.